src/Presentation/Startup.cs

```diff
@@ -1,21 +1,36 @@
 using System;
+using Data;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
 
 namespace Presentation
 {
+    
     public class Startup
     {
+        
+        public string ConnectionString { get; set; }
```

Comment:
Why is it public? Do we use it outside of `Startup`?

Reply:
Should we also consider removing this field? Wouldn't be easier to use a local variable instead?